README.md

```diff
@@ -399,17 +399,16 @@ TODO-A: - Team is already subscribed to the package
 TODO-B: - Team is not yet, but will subscribe to the package before promotion
 
 RULE: - Responsibilities implied by static builds promoted to main, which is
-RULE:   not a recommended but a common case with golang packages.
-RULE:   - the security team will track CVEs for all golang packages in main
+RULE:   not a recommended but a common case with golang and rust packages.
+RULE:   - the security team will track CVEs for all vendored/embedded sources in main
 RULE:   - the security team will provide updates to main for all `golang-*-dev`
 RULE:     packages
-RULE:   - the security team will provide updates to main for golang applications
-RULE:     whose non-vendored source code is affected as per normal procedures
-RULE:     (including e.g., sponsoring/coordinating uploads from teams/upstream
-RULE:     projects, etc)
+RULE:   - the security team will provide updates to main for non-vendored
+RULE:     dependencies as per normal procedures (including e.g.,
+RULE:     sponsoring/coordinating uploads from teams/upstream projects, etc)
 RULE:   - the security team will perform no-change-rebuilds for all packages
-RULE:     Built-Using `golang-*-dev` packages it has provided, and coordinate
-RULE:     testing with the owning teams responsible for the rebuilt packages
+RULE:     listing an CVE-fixed package as Built-Using and coordinate testing
```

Comment:
FYI, dh-golang has switched to using `Static-Built-Using:` for reporting the golang-*-dev packages that are used as build time dependencies; this has landed [in kinetic](https://launchpad.net/ubuntu/+source/dh-golang/1.55) and various golang leaf packages are starting to pick it up through rebuilds.

Reply:
Thanks @stevebeattie - that is interesting - from your need to track content that is contained - would that new `Static-Built-Using:` be equal or better to the former ask to get go.mod/go.sum/cargo.lock? So should we prefer to go towards asking for them instead - which due to the similarity with Debian should be easier to maintain.

Such usually hold packages - do you happen to know if/how vendored dependencies are listed in there?

Reply:
For reference, this was brought into dh-golang via
- https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921284
- https://salsa.debian.org/go-team/packages/dh-golang/-/commit/055ddf971b73fb20bf306d8faf7e4a31eb1b3928

dh-cargo has no such change and actually not any recent change, the current version is 7 months old.

Reply:
We discussed about this, not only does it not exist in the rust tools - it is only for the packaged dependencies and the whole "what if vendored" discussion is due to Ubuntu allowing special cases with vendoring - and those would not be covered by this.